docs/src/css/codes.css

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: CC-BY-NC-SA-4.0
+ */
+
+.codes .name {
+  cursor: alias;
+}
+
+.codes .name:hover {
+  background: var(--ifm-font-color-base-inverse);
+}
+
+.codes .name:active {
+  color: var(--ifm-font-color-base-inverse);
+  background: var(--ifm-font-color-base);
+}
+
+.codes table {
+  display: table;
+  font-size: 0.8em;
+  margin-bottom: 0;
+}
+
+.codes th,
+.codes td {
+  padding: 0.2rem;
+}
+
+.codes td {
+  position: relative;
+}
+
+.codes .behavior {
+  width: 1%;
+  min-width: 1rem;
+  cursor: help;
```

Comment:
This cursor should only be applied to the `a` tag. A simple `> a` should suffice.

Reply:
Thanks!

Reply:
Please could you the update?